frontend/src/components/Altimeter.js

```diff
@@ -5,43 +5,60 @@ import Button from "./Button";
 
 // #00e676 #00c853
 
-function Altimeter({telemetry}) {
+function Altimeter({telemetry, command}) {
+//Declare altitude variable outside to be able to restrain it to 2 decimals per
+    //has to parseFloat, otherwise gives undefined error with toFixed function.
+    let alt = telemetry === undefined ? "---" : (telemetry.enviro?.altitude);
+    let altDisplay = parseFloat(alt).toFixed(2);
```

Comment:
alt is eventually compared as a numerical value to determine if it is at a valid drop height. This leads to "---" being compared to the drop threshold which isn't good (string compared to number?). Instead let's deal with the initial situation where no data has been received yet by having the `canDrop()` return false until a packet has been received and altDisplay be "---" until a packet has been received. This can be achieved by using a bool that will act as a flag for "first_packet_received". As soon as enviro isn't undefined, set the flag to true. Then, use the flag in `canDrop()` and altDisplay